lib/CMakeLists.txt

```diff
@@ -18,10 +18,15 @@ set(PROJECT_SOURCES
     src/values/abstract_config_value.cc
     src/values/config_boolean.cc
     src/values/config_number.cc
-    src/values/config_int.cc
```

Comment:
In the Java code they have both ConfigInt and ConfigLong. ConfigInt returns its value as a long, unless shoved through ConfigNumber's intValueRangeChecked() method. The Ruby only has config_int, because of the way numbers are handled in Ruby. I decided to only implement config_long, because almost everywhere, they seem to just assume the value is a long anyway. 

However, not opposed to re-adding config_int.